### PR TITLE
Feature/buktopuha-leaderboard

### PR DESCRIPTION
**Pull request by C-Pro**
Like `/hussars` for `/roll` but `/znatoki` for  `/buktopuha`.

### CODE REVIEW

File: bot/skills/buktopuha.py [hunk 1]
```diff
@@ -29,6 +37,78 @@
...
+    """
+    BuKToPuHa document:
+    {
+        _id: 420,                                   # int       -- tg user id
+        meta: {...},                                # Dict      -- full tg user object (just in case)
+        game_counter: 10,                           # int       -- number of games started
+        win_counter: 8,                             # int       -- number of games won
+        total_score: 100,                           # int       -- total score gained
+        created_at: datetime(...),                  # DateTime  -- user record creation time
+        updated_ad": datetime(...)                  # DateTime  -- last record update time
```
Comment (egregors) [inline bot/skills/buktopuha.py:50]: Typo in comment (minor)

File: bot/skills/buktopuha.py [hunk 2]
```diff
@@ -305,3 +408,157 @@ def start_buktopuha(update: Update, context: CallbackContext):
...
+JPEG = "JPEG"
+EXTENSION = ".jpg"
+COLOR = "white"
+MODE = "L"
+FONT_SIZE = 12
+ZNATOKI_LIMIT_FOR_IMAGE = 25
+FONT = "firacode.ttf"
+
+
+def _create_empty_image(image_path, limit):
```
Comment (egregors) [inline bot/skills/buktopuha.py:515]: All this stuff looks like a copy-paste. Is it from `top`? 
Maybe we should extract it in some sort of common utility? 
I mean text-to-image func, zum Beispiel.

**Review by egregors — APPROVED** [review]
LGTM, but I don't like text-to-image duplication.

File: bot/skills/buktopuha.py (hunk 2 above)
Comment (C-Pro) [reply bot/skills/buktopuha.py:515]: It is from roll. Not common enough for me to justify abstracting away. I usually separate the code when my fingers are tired from copy-pasting.

File: bot/skills/buktopuha.py (hunk 1 above)
Comment (C-Pro) [reply bot/skills/buktopuha.py]: fixed

**Pull request merged**